ElementX/Sources/Services/Notification/Manager/UserNotificationCenterProtocol.swift

```diff
@@ -22,4 +22,8 @@ protocol UserNotificationCenterProtocol: AnyObject {
     func add(_ request: UNNotificationRequest) async throws
     func requestAuthorization(options: UNAuthorizationOptions) async throws -> Bool
     func setNotificationCategories(_ categories: Set<UNNotificationCategory>)
+    func getAuthorizationStatus() async -> UNAuthorizationStatus
```

Comment:
Apple suggests not to use `get` for async functions, only for methods with completion handlers :)
```suggestion
    func authorizationStatus() async -> UNAuthorizationStatus
```

Reply:
done.